salt/platform/win.py

```diff
@@ -0,0 +1,1171 @@
+# -*- coding: utf-8 -*-
+'''
+Windows specific utility functions, this module should be imported in a try,
+except block because it is only applicable on Windows platforms.
+
+
+Much of what is here was adapted from the following:
+
+    https://stackoverflow.com/a/43233332
+    http://stackoverflow.com/questions/29566330
+'''
+from __future__ import absolute_import, unicode_literals
+import os
+import collections
+import logging
+import psutil
+
+import ctypes
+from ctypes import wintypes
+
+from salt.ext.six.moves import range
+from salt.ext.six.moves import zip
+
+import win32con
+import win32con
+import win32api
+import win32process
+import win32security
+import win32service
+import ntsecuritycon
```

Comment:
Consider importing these are using these constants, rather than defining your own (assumed)   
```
 import pywintypes
 import winerror
```

Reply:
I'd like to get this merged. I'm committed to a lot of other work for the Fluorine release. If needed, let's make a separate issue about cleaning up the constants and assign it to me. I'll tackle that as time permits.